util/util.go

```diff
@@ -0,0 +1,59 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bmatcuk/doublestar"
+)
+
+func MustMkdirInCurrentDirectory(dirname string) {
+	dir := MustGetCurrentDir()
+	p := filepath.Join(dir, dirname)
+	MustMkdir(p)
+}
+
+func MustMkdir(dirname string) {
+	err := os.Mkdir(dirname, 0700)
+	if err != nil {
+		panic(err)
```

Comment:
Would be nice to have idempotency, panicing prevents further steps to be performed.
At least don't panic if `os.IsExist(err)`.